Assets/Scripts/Snowball Scripts/SnowInventory.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+/// <summary>
+/// Manages the player's snowball inventory. Stored on each player.
+/// </summary>
+public class SnowInventory : MonoBehaviour
+{
+    public int currentAmmo = 5;
+    private string name;
```

Comment:
the string 'name' is to generic for unity and is getting confused with another built in variable, i would suggest renaming this to something like penguinName for clarification

Reply:
What an excellent suggestion for the name! I have changed it to that.

Also, regarding your warning message, I also get one, but it was talking about the variable name, not the URP message you had.